controllers/storagecluster/cephcluster.go

```diff
@@ -1150,32 +1150,35 @@ func createPrometheusRules(r *StorageClusterReconciler, sc *ocsv1.StorageCluster
 	applyLabels(getCephClusterMonitoringLabels(*sc), &prometheusRule.ObjectMeta)
 
 	replaceTokens := []replaceToken{
-		{
-			recordOrAlertName: "CephMgrIsAbsent",
-			wordToReplace:     "openshift-storage",
-			replaceWith:       sc.Namespace,
-		},
+		createReplaceToken("", "CephMgrIsAbsent", "openshift-storage", sc.Namespace, ExprRuleSection, ""),
 	}
 
+	const DescriptionKey = "description"
 	// if nearFullRatio/backfillFullRatio/fullRatio are specified on the StorageCLuster CR, replace the values in the prometheus rule accordingly
 	specifiedNearFullRatio := sc.Spec.ManagedResources.CephCluster.NearFullRatio
 	specifiedBackfillFullRatio := sc.Spec.ManagedResources.CephCluster.BackfillFullRatio
 	specifiedFullRatio := sc.Spec.ManagedResources.CephCluster.FullRatio
 
 	if specifiedNearFullRatio != nil {
 		replaceTokens = append(replaceTokens,
-			createReplaceToken("", "", "75%", fmt.Sprintf("%.2f%%", *specifiedNearFullRatio*100)),
-			createReplaceToken("", "", "0.75", fmt.Sprintf("%f", *specifiedNearFullRatio)))
+			createReplaceToken("cluster-utilization-alert.rules", "CephClusterNearFull",
+				"75%", fmt.Sprintf("%0.0f%%", *specifiedNearFullRatio*100), AnnotationRuleSection, DescriptionKey),
+			createReplaceToken("cluster-utilization-alert.rules", "CephClusterNearFull",
+				"0.75", fmt.Sprintf("%0.2f", *specifiedNearFullRatio), ExprRuleSection, ""))
 	}
 	if specifiedBackfillFullRatio != nil {
 		replaceTokens = append(replaceTokens,
-			createReplaceToken("", "", "80%", fmt.Sprintf("%.2f%%", *specifiedBackfillFullRatio*100)),
-			createReplaceToken("", "", "0.80", fmt.Sprintf("%f", *specifiedBackfillFullRatio)))
+			createReplaceToken("cluster-utilization-alert.rules", "CephClusterCriticallyFull",
+				"80%", fmt.Sprintf("%0.0f%%", *specifiedBackfillFullRatio*100), AnnotationRuleSection, DescriptionKey),
```

Comment:
80%/0.80 needs to replaced in CephClusterCriticallyFull & CephOSDCriticallyFull.

Reply:
Same question as above, but regarding `CephOSDCriticallyFull` alert, are we going to use the same `*BackfillFullRatio` value or create new fields for CephOSD* alerts?